local-setup/nginx/sites-enabled/odk.conf

```diff
@@ -34,7 +34,7 @@ server {
   location / {
     proxy_pass              http://odk.aether.local:8002;
 
-    proxy_set_header        Host               $host;
+    proxy_set_header        Host               $host:8443;
```

Comment:
This is addressed in another PR, but I guess it's OK to leave it here since that other one has been merged

Reply:
This change comes from your commit :wink: 

(nightmare in the commits history :scream: )